package.json

```
@@ -54,39 +52,45 @@
     "preload-fonts": "gatsby-preload-fonts",
     "develop": "gatsby develop",
     "format": "pretty-quick --staged",
-    "test": "jest",
+    "test": "jest --env=jsdom",
+    "test:update-snapshots": "jest --env=jsdom -u",
     "lint": "eslint src",
     "test:coverage": "jest --env=jsdom --coverage",
-    "test:watch": "jest --watch",
+    "test:watch": "jest --env=jsdom --watch",
     "view:coverage": "open coverage/lcov-report/index.html",
     "prepare": "husky install"
   },
   "devDependencies": {
-    "@babel/core": "7.14.2",
-    "@testing-library/jest-dom": "5.12.0",
-    "@testing-library/react": "11.2.6",
-    "babel-eslint": "10.1.0",
-    "babel-jest": "26.6.3",
-    "babel-preset-gatsby-package": "1.5.0",
+    "@babel/core": "7.16.12",
```

One wonders if using semantic versioning on the major version would be smart here. i.e.,
```suggestion
    "@babel/core": "^7.16.12",
```